src/features.ts

```diff
@@ -9,5 +9,9 @@ export interface Features {
 }
 
 export const CurrentFeatureSet: Features = {
+  REDmodAutoconvertArchives: Feature.Enabled,
+};
+
+export const DefaultFeatureSetForTesting: Features = {
   REDmodAutoconvertArchives: Feature.Disabled,
 };
```

Comment:
These should be congruous. The production features set and testing feature set shouldn't be different, imo. and just set the feature as disabled for the old style archive install

Reply:
> I don't think we should have the feature be enabled by default without a settings UI to disable it or not.

It's the only way to test it for now.

I'm not sure what the best approach is (especially if the features are togglable, which means there is no prod features set as such), but there has to be a baseline for tests, or then every test has to be updated to specify the feature set it uses. Or if we switch the baseline to enabled conversion, then there's ~25 tests that need to be updated.

Reply:
hmm.... ok. I guess if this isn't the nexus upload it is okay... fair point on the tests though.